samcli/hook_packages/terraform/hooks/prepare.py

```diff
@@ -249,12 +249,35 @@ def _translate_to_cfn(tf_json: dict, output_directory_path: str, terraform_appli
         _enrich_mapped_resources(
             sam_metadata_resources, cfn_dict["Resources"], output_directory_path, terraform_application_dir
         )
+
+        LOG.debug("Generate custom makefile for building Lambda resources")
+        _generate_custom_makefile(sam_metadata_resources, cfn_dict["Resources"], output_directory_path)
```

Comment:
nit: lets use .get() whenever we are accessing dict keys just so we dont run into key errors.

Reply:
done